extensions-core/azure-extensions/src/main/java/org/apache/druid/storage/azure/AzureTaskLogsConfig.java

```diff
@@ -37,19 +36,14 @@ public class AzureTaskLogsConfig
   @NotNull
   private String prefix = null;
 
-  @JsonProperty
-  @Min(1)
-  private int maxTries = 3;
```

Comment:
i don't really see this config documented anywhere and its not supported with S3TaskLogs or GoogleTaskLogs so i just removed it